examples/main/modules.tf

```diff
@@ -83,9 +96,12 @@ module "network_security_group" {
   networK_watcher_resource_group_name = data.azurerm_network_watcher.network_watcher.resource_group_name
 
   flow_log_retention_policy_enabled = true # default to true
-  flow_log_retention_policy_days    = 7    # default to 7
+  flow_log_retention_policy_days    = 91   # default to 91
 
-  flow_log_storage_account_id                    = module.logs.logs_storage_account_id
+  # Make sure to use a storage account with no existing lifecycle management rules
```

Comment:
Can you add also this note into README?

Reply:
done,

Reply:
Don't you generate the README? if not I'll add the new var to the README